gamify-specs/src/test/java/ch/heigvd/gamify/api/spec/defs/BadgesDefs.java

```diff
@@ -0,0 +1,86 @@
+package ch.heigvd.gamify.api.spec.defs;
+
+import ch.heigvd.gamify.ApiException;
+import ch.heigvd.gamify.api.BadgesApi;
+import ch.heigvd.gamify.api.dto.Badge;
+import ch.heigvd.gamify.api.spec.env.Environment;
+import io.cucumber.java8.En;
+
+public class BadgesDefs implements En
+{
+    public BadgesDefs(Environment environment)
+    {
+        Given("I create the badge {word} linked to category {word}", (String badge, String category) ->
+                environment.getClient().putPayload(badge, newBadge(badge, category)));
+
+        When("I PUT the {word} badge to the api.badges endpoint", (String badgeName) ->
```

Comment:
```suggestion
        When("I PUT the {word} badge to the api.badges.*** endpoint", (String badgeName) ->
```

This way, it's clear that it's not put at `/badges` but at `badges/something`. We use the same scheme for other Cucumber steps.